lib/presentation/widget/destination_picker/destination_picker_widget.dart

```diff
@@ -354,4 +354,14 @@ class _DestinationPickerWidgetState extends State<DestinationPickerWidget> {
       style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
     );
   }
+
+  bool _isDestinationAvailable(DestinationType destinationType) {
+    if (_destinationPickerArguments != null
+        && _destinationPickerArguments.availableDestinationTypes != null
+        && _destinationPickerArguments.availableDestinationTypes.contains(destinationType)
+    ) {
+      return true;
+    }
+    return false;
```

Comment:
Just asking, is returning `_destinationPickerArguments != null
        && _destinationPickerArguments.availableDestinationTypes != null
        && _destinationPickerArguments.availableDestinationTypes.contains(destinationType)` the same ?